### PR TITLE
Added a min width of 320px to Message thread to render file cards completely in mobile view

### PR DESCRIPTION
**Pull request by prabhjot-msft**
# What
Added a min width of 320px to Message thread to render file cards completely in mobile view

# Why
https://skype.visualstudio.com/SPOOL/_workitems/edit/2806013

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/react-components/src/components/styles/MessageThread.styles.ts [hunk 1]
```diff
@@ -31,7 +31,8 @@ export const messageThreadContainerStyle = mergeStyles({
   maxHeight: '100%',
   overflow: 'auto',
   position: 'relative',
-  alignSelf: 'center'
+  alignSelf: 'center',
+  minWidth: '320px'
```
Comment (JamesBurnside) [inline packages/react-components/src/components/styles/MessageThread.styles.ts:35]: Our current minWidth for the chat composite is 280px:
image
https://60c7ae6891f0e90039d7cd54-actiqxaqcq.chromatic.com/?path=/docs/composites-chat-basicexample--basic-example

Can we reduce the space the file cards need?

**Review by JamesBurnside — CHANGES_REQUESTED** [review]

File: packages/react-components/src/components/styles/MessageThread.styles.ts (hunk 1 above)
Comment (anjulgarg) [reply packages/react-components/src/components/styles/MessageThread.styles.ts:35]: @pereiralex thoughts?

Comment (anjulgarg) [reply packages/react-components/src/components/styles/MessageThread.styles.ts:35]: This dimension came from AlexP.
@prabhjot-msft let's reach out to AlexP and get his opinion here.

Comment (JamesBurnside) [reply packages/react-components/src/components/styles/MessageThread.styles.ts:35]: note: for context this was reduced to support the galaxy fold

**Pull request closed without merging**